bin/sync_slurm_acct.py

```diff
@@ -112,7 +114,7 @@ def main():
         sacctmgr_commands += slurm_institute_accounts(slurm_account_info, clusters)
 
         # All users belong to a VO, so fetching the VOs is necessary/
-        account_page_vos = [mkVo(v) for v in client.vo.get()[1]]
+        account_page_vos = [mkVo(v) for v in client.vo.institute[opts.options.institute].get()[1]]
```

Comment:
that should strictly not be needed I believe.

If no institute is given, the institute of the requesting user is used.

Reply:
```python

    def get_queryset(self):
        """
        Get the information of all the VOs
        """
        institute = self.kwargs.get('institute', self.request.user.person.institute.name)
        return VirtualOrganisation.objects.filter(institute__name=institute)
```